.github/workflows/ci.yml

```diff
@@ -28,6 +28,8 @@ jobs:
       uses: actions/setup-node@v1
       with:
         node-version: ${{ matrix.node }}
+    - name: Setup Bun
+      uses: oven-sh/setup-bun@v1
```

Comment:
It would be better to just add bun as a dev dep to `which-pm-runs` as that way it will work for everyone locally without additional configuration.